packages/bot/src/assets/sass/scratch/flyout.scss

```diff
@@ -16,6 +16,14 @@
     box-shadow: 0 2px 8px 0 $COLOR_LIGHT_BLACK_2;
     min-width: 400px;
 
+    &__item:hover {
+        .flyout__button-add-hide {
+            display: block;
+        }
+        .flyout__item-info {
```

Comment:
Should we underline `Learn more` when hovering over the block? In my opinion it looks a bit off.

Reply:
i just wanted to make it more prominent as i think it's not enough, let's please discuss again when it's in staging? it's an easy fix, so won't steel time

Reply:
i removed underlined style for the learn more link